biomage/account/account.py

```diff
@@ -267,6 +277,87 @@ def create_users_list(user_list, header, input_env, aws_profile, overwrite):
             out.write("%s,%s,%s\n" % (full_name, email, password))
 
 
+@click.command()
+@click.option(
+    "--user_list",
+    required=True,
+    help="User list containing user and email for the new accounts in csv.",
+)
```

Comment:
You are passing the name of a file here, right? I would suggest mentioning it in the description or the name of this param somehow